src/user-picker/UserPickerCard.js

```diff
@@ -41,9 +42,9 @@ const getUsersToShow: Selector<User[]> = createSelector(
 );
 
 export default function UserPickerCard(props: Props): Node {
-  const { filter } = props;
+  const { filter, showOwnUser } = props;
 
-  const users = useSelector(getUsersToShow);
+  const users = useSelector(showOwnUser ? getUsers : getUsersToShow);
```

Comment:
If we add a case here where `getUsersToShow` is not used, then `getUsersToShow`'s name becomes inaccurate, as well as the following comment on it:

```
// The users we want to show in this particular UI.
```

Reply:
My first thought was to change the name of `getUsersToShow` to `getUsersExceptSelf` or something better and shorter, then create a function that takes showOwnUser as a parameter if true returns`getUsersExceptSelft` otherwise return `getUsers`.
and add the comment you mentioned above the function

Reply:
@gnprice, is the `createSelector` in `getUsersToShow` an instance of #3927?